polaris-react/src/components/Tooltip/tests/Tooltip.test.tsx

```diff
@@ -98,13 +97,110 @@ describe('<Tooltip />', () => {
     );
 
     findWrapperComponent(tooltip)!.trigger('onKeyUp', {
-      keyCode: Key.Escape,
+      key: 'Escape',
     });
     expect(tooltip).toContainReactComponent(TooltipOverlay, {
       active: false,
     });
   });
 
+  it('calls onVisibilityChange when initially activated', () => {
```

Comment:
```suggestion
  it('does not call onVisibilityChange when initially activated', () => {
```
I realized I wrote this test for when the `active` prop is set to `true` (which forces the tooltip to be active when first rendered), but didn't set the `useEffect` up to call `onVisibilityChange` if the tooltip initially renders as activated. 

If you think it'd be useful for that case, I have a commit stashed that I can push up to support that. Otherwise, the test just needs to be updated to `it ('does not ...)`

Reply:
The test passes and it is called with false when it closes. This test looks correct to me unless I'm missing something

Reply:
The expect clauses are correct but the sentence isn't--the expect clauses test that the callback is not fired, but the test says it is fired.

Reply:
Ignore me lol, the test was right and the expect clauses were wrong--the callback does get called if forced active on initial render!